### PR TITLE
feat: add scripts/install.sh

### LINKED ISSUE #11: Allow a "curl" one line installation
A lot of packages allow you to install them by pasting a "one-liner" into the terminal with curl, something like `curl https://install.seam.co | sh`. They do this by serving a shell file at the url. How can we create that shell file?

### PR DESCRIPTION
**Pull request by mikewuu**
closes #11 

- fixed to release 0.0.8. We should probably bump the version automatically, bundle the install script in a release? 

### to give it a go

```
curl 'https://raw.githubusercontent.com/seamapi/seam-cli/install-script/scripts/install.sh' | sudo sh
```

### CODE REVIEW

**Review by seveibar — APPROVED** [review]

File: scripts/install.sh [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+      error="${error} Please try again."
+    fi
+
+    log ""
+    log "$error"
+
+    if [ "$status_code" -eq 404 ]; then
+      log ""
+      log "Please report this issue:"
+      log "https://github.com/DopplerHQ/cli/issues/new?template=bug_report.md&title=[BUG]%20Unexpected%20404%20using%20CLI%20install%20script"
```
Comment (razor-x) [inline scripts/install.sh:229]: Why is doppler here?

**Comment by razor-x** [comment]
@mikewuu If this was adapted from https://github.com/DopplerHQ/cli/blob/08cd5113f646ed98d4ca806c204428c68a69b8a1/scripts/install.sh then we need to adhere by their license: https://github.com/DopplerHQ/cli/blob/master/LICENSE

In particular, the requirements of section 4 need to be met. Since it's just one file, we can just put the license with the notice we changed the file with a link back to the source as a comment in the install script.

**Comment by mikewuu** [comment]
> @mikewuu If this was adapted from https://github.com/DopplerHQ/cli/blob/08cd5113f646ed98d4ca806c204428c68a69b8a1/scripts/install.sh then we need to adhere by their license: https://github.com/DopplerHQ/cli/blob/master/LICENSE
> 
> In particular, the requirements of section 4 need to be met. Since it's just one file, we can just put the license with the notice we changed the file with a link back to the source as a comment in the install script.

Addressed in https://github.com/seamapi/seam-cli/pull/65

File: scripts/install.sh (hunk 1 above)
Comment (mikewuu) [reply scripts/install.sh:229]: Because it was adapted from their script. Removed in https://github.com/seamapi/seam-cli/pull/65

**Pull request merged**